.github/workflows/ci.yaml

```diff
@@ -352,24 +352,33 @@ jobs:
           path: |
             test/e2e/node_modules
           key: ${{ runner.os }}-${{ env.NODE_VERSION }}-${{ hashFiles('test/e2e/package-lock.json') }}
+      - name: Retrieve cached photofinish binary
+        uses: actions/cache@v3
+        id: photofinish-cache
+        with:
+          path: /tmp/photofinish
+          key: ${{ runner.os }}-photofinish
       - name: Check Eslint and JS Code Format
         run: cd test/e2e && npm run lint && npm run format:check
       - name: Mix setup
         run: mix setup
       - name: Run trento detached
         run: mix phx.server &
-      - name: Get Photofinish for e2e testing
-        run: |
-          set -x
-          wget -P /tmp https://github.com/trento-project/photofinish/releases/download/v1.1.0/photofinish
-          chmod +x /tmp/photofinish
+      - name: Install photofinish
+        uses: jaxxstorm/action-install-gh-release@v1.5.0
+        with:
+          repo: trento-project/photofinish
```

Comment:
We should still pin the binary to a version here (`1.2.0`) and the action itself already have a caching system, using the `cache: enable` entry